### ISSUE TITLE
Irrelevant desktop tooltip of Processing 4.4.4 on Windows

### ISSUE THREAD

**Issue by yangtze314** [processing/processing4#1122]
### Most relevant area for this enhancement?

Accessibility

### Feature to enhance

Tooltip of application shortcut on desktop

### Desired enhancement

When hovered on my desktop, shortcuts of Processing 4.4.4 display the tooltip "My Application Description" (likely a default value). I believe that this should be changed to something relevant to the application, such as "Processing, an open source project maintained by the Processing Foundation".

Although I am not sure if this feature enhancement should be categorized under "Accessibility", I did so because text within tooltips may be utilized by accessibility programs.

### Potential benefits

A relevant tooltip can enhance accessibility for disabled users.

### Possible drawbacks

None.

### Additional context

Screenshot of a shortcut displaying the tooltip:
![Image](https://github.com/user-attachments/assets/1e79c479-7a6e-41de-9351-1ea07746b648)

### Would you like to work on the issue?

No

**Comment by SableRaf** [comment]
Hi @yangtze314, thank you for catching this and taking the time to report it!

The issue seems to come from this line in the installer configuration:

https://github.com/processing/processing4/blob/1ee243c93a939e466f1f5ed39cff4238e0a37f32/app/windows/Processing.wxs#L27

If you're curious about how this could be updated and would like to try making the change, I’d be happy to guide you through the steps. No pressure at all, just let me know!

**Comment by yangtze314** [comment]
Hello @SableRaf. Since I have not yet familiarized myself with the GitHub workflow, and also because I am currently occupied by an upcoming exam, I think I should leave this change for more experienced people. Nonetheless, thank you for your offer! 😊

**Comment by thisizaro** [comment]
Hello @SableRaf . I believe I might be able to improve the description. Would you be open to me giving it a try and submitting a PR? Let me know if you're okay with that, I'd be happy to help!

**Comment by SableRaf** [comment]
Hi @thisizaro and thanks for offering to help! I've assigned the issue to you. Please go ahead and make a PR.

**Comment by thisizaro** [comment]
Hi @SableRaf I made a [pull request](https://github.com/processing/processing4/pull/1127) after solving the issue. Please check.
